lynx/common/square.py

```diff
@@ -1,38 +1,37 @@
 from dataclasses import dataclass, field
-from typing import List, NoReturn
+from typing import List, Optional
 
 from lynx.common.object import Object
 
 
-# Class wraping logic of ground and objects placed on it
+# class wrapping logic of a tile and objects placed on it
 @dataclass
 class Square:
-    ground: Object = None
+    tile_object: Optional[Object] = None
```

Comment:
I preferred ground name, but up to ur experience

Reply:
I agree with @EasyJB

Reply:
"ground" is not telling what this one `Object` should always be (it should always be an `Object` with `tile` tag i.e. "tile_object")
Also, I would not want to introduce names that can be confused with "tile" or "square".